src/vs/workbench/electron-sandbox/shared.desktop.main.ts

```diff
@@ -263,7 +263,7 @@ export abstract class SharedDesktopMain extends Disposable {
 		]);
 
 		// Workspace Trust Service
-		const workspaceTrustManagementService = new WorkspaceTrustManagementService(configurationService, storageService, uriIdentityService, configurationService, environmentService);
+		const workspaceTrustManagementService = new WorkspaceTrustManagementService(configurationService, environmentService, storageService, uriIdentityService, configurationService);
```

Comment:
Ideally we keep service registration in the top level entry files (e.g. [here](https://github.com/microsoft/vscode/blob/73c85159e72069f69db46fad178f37b5d015f5bc/src/vs/workbench/workbench.common.main.ts#L96-L96)) unless one of the core services depend on them.

Reply:
See comment above.